docs/content/src/learn/tutorials/saas-integrations/gmail/using-the-gmail-connector/README.md

```diff
@@ -1,138 +1,125 @@
 5. Click **Create**. Your client ID and client secret will appear.
-6. In a separate browser window or tab, visit [OAuth 2.0 playground](https://developers.google.com/oauthplayground). Click on the _OAuth 2.0 configuration_ icon in the top right corner and click on **Use your own OAuth credentials** and provide your _OAuth Client ID_ and _OAuth Client secret_.
+6. In a separate browser window or tab, visit [OAuth 2.0 playground](https://developers.google.com/oauthplayground). Click on the _OAuth 2.0 configuration_ icon in the top right corner and click on **Use your OAuth credentials** and provide your _OAuth Client ID_ and _OAuth Client secret_.
 7. Select the required Gmail API scopes from the list of APIs, and then click **Authorize APIs**.
 8. When you receive your authorization code, click **Exchange authorization code for tokens** to obtain the refresh token and access token.
 9. You can enter the credentials in the HTTP client config when defining the service.
 
-#### Adding Gmail configuration
-
-In the Health Care Service we created in previous tutorials, we can add an HTTP client config for Gmail as below.
-
-<!-- INCLUDE_CODE_SEGMENT: { file: src/tutorial/health_care_service.bal, segment: segment_1 } -->
+Create a project.
+```bash
+ballerina new using-the-gmail-connector
+```
+Navigate to the project directory and add a module using the following command.
+```bash
+ballerina add gmail_client_application
+```
 
-Then we can create the Gmail client using the above config.
+Add a `ballerina.conf` file and create .bal files with meaningful names as shown in the project structure given below.
+```
+using-the-gmail-connector
+├── Ballerina.toml
+├── ballerina.conf
+└── src
+    └── gmail_client_application
+        ├── Module.md
+        ├── gmail_client.bal
+        ├── resources
+        └── tests
+            ├── resources
+            └── gmail_client_test.bal
+```
 
-<!-- INCLUDE_CODE_SEGMENT: { file: src/tutorial/health_care_service.bal, segment: segment_2 } -->
+Update the `ballerina.conf` file with the token configuration required for accessing the Gmail account.
+```
+ACCESS_TOKEN = ""
+CLIENT_ID = ""
+CLIENT_SECRET = ""
+REFRESH_TOKEN = ""
+```
 
-#### Generating mail body
+You can open the project with VS Code. The integration implementation is written in the `gmail_client.bal` file.
 
-We can use a util function to generate the mail body, based on the response received from the payment endpoint.
+The code segment given below can be used to create a Gmail client. Ballerina Integrator VS Code plugin contains a snippet for the same, which can be loaded using the autocomplete feature for the `client/gmail` keyword.
 
-<!-- INCLUDE_CODE_SEGMENT: { file: src/tutorial/health_care_service.bal, segment: segment_3 } -->
+```ballerina
```

Comment:
Shall we add this as a code segment, since this is included in the code file.
Refer [this](https://github.com/IrushiL/ballerina-integrator/tree/gmailGuide/docs/doc-generator#b-include-a-code-segment)

Reply:
Fixed in https://github.com/wso2/ballerina-integrator/pull/416